### PR TITLE
Fix common-files linking inside containers

### PR DESCRIPTION
**Pull request by pawelgrzybek**
## What does this implement/fix? Explain your changes.

This MR fix common-files linking inside Docker containers. Very good spot @israelboudoux thank you for collaboration.

### CODE REVIEW

**Review by israelboudoux — APPROVED** [review]

**Review by daveroga — APPROVED** [review]
LGTM

**Review by signorecello — APPROVED** [review]

**Comment by israelboudoux** [comment]
Hey my friend @pawelgrzybek, sorry, I had to do some tweaks in this PR, and I did a workaround since things weren't going well when we do the last `npm link`. I inspected the docker containers and I realised that something very weird happens when we do the `npm link` before or after the `npm ci`. 
I'll use the `nightfall-client` app as an example...

Inside the docker container...

when doing the `npm link` BEFORE `npm ci`
```
/app/node_modules - 1780 folders
/app/node_modules/@polygon-nightfall/common-files/node_modules - 1 folder
```
when doing the `npm link` AFTER `npm ci`
```
/app/node_modules - 1632 folders
/app/node_modules/@polygon-nightfall/common-files/node_modules - +1.8k folders
```

For me, these differences make no sense since it is the same package, the `common-files`. Another point is that the `npm link` changes the `node_modules` app's folder, which for me, on top of my ignorance isn't correct. 

Then my workaround was to copy the folders (except the `node_modules`) to `/app/node_modules/@polygon-nightfall/common-files/`.

File: docker/worker.Dockerfile [hunk 1]
```diff
@@ -32,5 +32,7 @@ COPY ./zokrates-worker/start-dev ./start-dev
 
 RUN npm link @polygon-nightfall/common-files
 RUN npm ci
+COPY common-files/classes common-files/utils common-files/constants node_modules/@polygon-nightfall/common-files/
```
Comment (pawelgrzybek) [inline docker/worker.Dockerfile:35]: Same here 😔

File: docker/admin.Dockerfile [hunk 2]
```diff
@@ -24,5 +24,7 @@ COPY nightfall-administrator/docker-entrypoint.sh nightfall-administrator/packag
 
 RUN npm link @polygon-nightfall/common-files
 RUN npm ci
+COPY common-files/classes common-files/utils common-files/constants node_modules/@polygon-nightfall/common-files/
```
[the remark this answers is not in the document]
Comment (israelboudoux) [reply docker/admin.Dockerfile:27]: I know, but this "action" doesn't create any mess, it only sends folders that have code - no package*.json or node_modules are being copied.

**Comment by druiz0992** [comment]
Client liveness test is failing because currently testnet is down. I did not realize that this dependency existed. In any case, I’m fine accepting this Pr if only this test fails

**Pull request merged**